### ISSUE TITLE
Animate broken in Chrome?

### ISSUE THREAD

**Issue by Kirkman** [NUKnightLab/juxtapose#56]
I'm trying the plugin for the first time today as a replacement for old code I wrote several years ago. It looks good, but I wanted to point out a couple things about the animate functions:

* The glide-to-click-point animation works well for me in IE11-win and Firefox 33-win. But it does not work in Chrome 43.0.2357.132-win. Instead of gliding, the slider jumps to where I click. I see this issue when making my own slider locally, but I also see it on the example at https://juxtapose.knightlab.com/

* Related to the above, the example on https://juxtapose.knightlab.com/ is broken in IE11-win. When you move the slider, both images resize (one squishes, the other gets fatter) rather than re-crop.

Also, a feature request:

* Would you consider adding an option to animate the slider gliding to its start position from the left side, onload? Here's an old example of what I'm trying to describe: http://images.stltoday.com/mds/00003582/content/index.html

**Comment by duner** [comment]
I'm going to move the feature request into another issue, see #66.

**Comment by duner** [comment]
Looks like the problem with IE11 is that IE doesn't support `initial` in CSS. I *think* that the above commit should fix the problem, but we should probably test it in a few more browsers. It might be possible to just get rid of the `initial`s altogether and just leave the `none`s.

**Comment by duner** [comment]
Hi @Kirkman. I opened this up in Chrome 43 on Windows in Browserstack and was unable to replicate the problem — the gliding seemed to be working just fine. Are you still seeing this issue?

**Comment by Kirkman** [comment]
Yes, I am still seeing that behavior in Chrome Win 44.0.2403.157m, but it's intermittent.

However, I just spent 20 minutes digging into my Chrome extensions, and I think the problem may be related to an "autocomplete = on" extension. This extension is supposed to remove autocomplete from form fields so that I can save passwords in Chrome or 1Password.

For whatever it's worth, I also saw a new issue: the right edge of the photo wiggles or flickers during the glide transition. This happens even when I boot Chrome in safe mode (all extensions disabled). I'm not seeing this wiggle/flicker in Mac Chrome.
